onnxruntime/python/tools/tensorrt/perf/perf_utils.py

```diff
@@ -4,21 +4,32 @@
 import logging
 import coloredlogs
 import re
+import sys
 
 debug = False
 debug_verbose = False 
 
+def get_output(command):
+    p = subprocess.run(command, check=True, stdout=subprocess.PIPE)
+    output = p.stdout.decode("ascii").strip()
+    return output
+
 def find(regex_string): 
     import glob
     results = glob.glob(regex_string)
     results.sort()
     return results
 
+def pretty_print(pp, json_object):
+    pp.pprint(json_object)
+    sys.stdout.flush()
+
 def get_latest_commit_hash():
```

Comment:
the current dockerfile deletes the ort repo. Not sure how to get the latest commit/ looking into ways to do this (saving to dockerfile)